HISTORY.md

```diff
@@ -1,3 +1,10 @@
+3.1.1 / 2017-10-31
+==================
+
+  * Wrap initialize functions of integrations in try/catch statement.
+  * Add logging of failed initializations.
+  * Add a `failedInitializations` array to prototype to capture names of any failed integrations to pass back to Segment api as part of event `_metadata`.
```

Comment:
I might leave it at "Add a `failedInitializations` array to prototype to capture names of any failed integrations". 

a) The Segment part only applies if you're using our hosted version and not the open source version.
b) the component uploading to Segment is outside of this repo.